src/test_runner.cr

```diff
@@ -209,7 +209,14 @@ module Mint
       end
 
       get "/runtime.js" do
-        Assets.read("runtime.js")
+        if runtime_path = @flags.runtime
+          raise RuntimeFileNotFound, {
+            "path" => runtime_path,
+          } unless ::File.exists?(runtime_path)
+          ::File.read(runtime_path)
+        else
+          Assets.read("runtime.js")
+        end
```

Comment:
This could be loaded into memory as well, preferably within the local variable after the `page_source` definition.

Reply:
Done :)